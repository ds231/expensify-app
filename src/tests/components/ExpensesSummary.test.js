import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';
import expenses from '../fixtures/expenses';

test('should correctly count and total one expense', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={expenses[0].amount} />);
    expect(wrapper).toMatchSnapshot();
});

test('should correctly count and total two expenses', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={2} expensesTotal={expenses[0].amount + expenses[1].amount} />);
    expect(wrapper).toMatchSnapshot();
});