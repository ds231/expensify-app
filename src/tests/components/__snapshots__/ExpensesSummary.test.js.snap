// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly count and total one expense 1`] = `
<div>
  <p>
    Viewing 
    1
     
    expense
    , totalling 
    $1.95
  </p>
</div>
`;

exports[`should correctly count and total two expenses 1`] = `
<div>
  <p>
    Viewing 
    2
     
    expenses
    , totalling 
    $1,096.95
  </p>
</div>
`;
